assets/js/app/Panel.js

```diff
@@ -770,6 +770,58 @@ LocusZoom.Panel.prototype.generateExtents = function(){
 };
 
 /**
+ * Generate an array of ticks for an axis. These ticks are generated in one of three ways (highest wins):
+ *   1. An array of specific tick marks
+ *   2. Query each data layer for what ticks are appropriate, and allow a panel-level tick configuration parameter
+ *     object to override the layer's default presentation settings
+ *   3. Generate generic tick marks based on the extent of the data
+ * @param {('x'|'y1'|'y2')} axis The string identifier of the axis
+ * @returns {Number[]|Object[]}  TODO: number format?
+ *   An array of numbers: interpreted as an array of axis value offsets for positioning.
+ *   An array of objects: each object must have an 'x' attribute to position the tick.
+ *   Other supported object keys:
+ *     * text: string to render for a given tick
+ *     * style: d3-compatible CSS style object
+ *     * transform: SVG transform attribute string
+ *     * color: string or LocusZoom scalable parameter object
+ */
+LocusZoom.Panel.prototype.generateTicks = function(axis){
+
+    // Parse an explicit 'ticks' attribute in the axis layout
+    if (this.layout.axes[axis].ticks){
+        var layout = this.layout.axes[axis];
+
+        var baseTickConfig = layout.ticks;
+        if (Array.isArray(baseTickConfig)){
+            // Array of specific ticks hard-coded into a panel will override any ticks that an individual layer might specify
+            return baseTickConfig;
+        }
+        //  TODO: Didn't have time to incorporate the broader ticks.data mechanism of PR 112 into this release; this method can be expanded in future
```

Comment:
FYI PR 112 is migrating away from the `ticks.data` approach to something a bit closer to what we see in this PR.